web/src/public/index.js

```diff
@@ -119,52 +74,167 @@ function plotTravelMonths(select, onSelect) {
         .text(d => d);
 }
 
-function plotParks(svg, npd, projection, filters) {
-    const defaultRadius = 3.5;
-    const defaultColor = '#0026ff';
+function plotParks(svg, npd, scores, projection, filters) {
+    const defaultColor = '#4682b4';
+    const parkRadius = d3.scaleLog().range([2, 8]).domain(d3.extent(npd, d => d.area));
+    let tooltip = d3.select("body").append("div") // for image
+        .attr("class", "tooltip")
+        .style("opacity", 1e-6);
 
-    svg.selectAll('circle').remove();
```

Comment:
The remove was here to totally refresh the DOM so when the circles are re-drawn - there are no collisions.
Reasons for removing this?

Reply:
put it back